test/integration/infrastructure/cleanup.go

```diff
@@ -0,0 +1,76 @@
+// Copied from https://github.com/kubernetes/kubernetes/blob/v1.19.0-beta.0/test/e2e/framework/cleanup.go
```

Comment:
/invite @msohn 
for coping this file

Reply:
You can update this to something like this - https://github.com/gardener/machine-controller-manager/blob/master/pkg/controller/deployment.go#L1-L20. @msohn had suggested such changes earlier for MCM.

Reply:
there is already a copy of this filein the gardener test framework: https://github.com/gardener/gardener/blob/master/test/framework/cleanup.go .
You could reuse the this one.

Reply:
Yep, I have to update it as in this file recently is updated with a fix in the upstream that the test uses/relies on - https://github.com/kubernetes/kubernetes/commit/708261e06c754ed9f757e1c91fd12e446857b702 (to execute the cleanup actions in orderer manner (reverse order of their addition).